api/src/main/java/com/doordeck/sdk/jackson/Jackson.java

```diff
@@ -40,7 +38,6 @@ private Jackson() { /* static class */ }
     public static final ObjectMapper OBJECT_MAPPER = new ObjectMapper()
             .registerModule(new Jdk8Module())
             .registerModule(new OptionalUpdateModule())
```

Comment:
You may need to add `JavaTimeModule` here but given we're using custom ser/deser let's leave this for another time, we should be able to remove those custom ser/deser in favour of things like @JsonShape, @JsonFormat, etc

Reply:
Take a look, about to push